SDVModTest/UIElements/ExperienceBar.cs

```diff
@@ -224,22 +270,22 @@ private void OnPreRenderHudEvent(object sender, EventArgs e)
                     {
                         ShowExperienceBar();
                         if (experienceEarnedThisLevel - _currentExperience > 0 &&
-                        _showExperienceGain)
+                        _showExperienceGain.IsOn)
                         {
                             _experiencePointDisplays.Add(
-                                new ExperiencePointDisplay(
-                                    experienceEarnedThisLevel - _currentExperience,
-                                    Game1.player.getLocalPosition(Game1.viewport)));
+                                    new ExperiencePointDisplay(

```

Comment:
I honestly think this is too much tabbing. You're running to the right really fast like this.

Reply:
yes it is. Auto formatting issues.